Arduino_package/hardware/libraries/SPI/SPI.cpp

```diff
@@ -43,7 +43,7 @@ SPIClass::SPIClass(void *pSpiObj, int mosi, int miso, int clk, int ss)
 
     pinUserSS = -1;
 
-    defaultFrequency = 20000000;
+    defaultFrequency = 2000000;
```

Comment:
AmebaD Provide two SPI ports
SPI0 (High speed): configured as master or slave with Max. Baud rate: 50MHz.
SPI1 (Normal speed): configured as master with Max. Baud rate: 25MHz.

Does the board need 2M as default? if Yes, Could we add a marco instead of change it.